surface_dynamics/flat_surfaces/origamis/origami_dense.pyx

```diff
@@ -2069,7 +2074,6 @@ cdef class Origami_dense_pyx:
         if self._n != other._n:
             return False
         cdef Origami_dense_pyx ss, oo
-        cdef ms, mo
```

Comment:
@videlec was there a reason for this `cdef`?

Reply:
I think that I always prefer declaring my variables. Nothing more.